src/rbx/datastore.rs

```diff
@@ -244,23 +244,18 @@ async fn handle_res_ok(res: Response) -> Result<(), Error> {
 
 fn build_url(endpoint: &str, universe_id: UniverseId) -> String {
     if endpoint.is_empty() {
-        format!(
-            "https://apis.roblox.com/datastores/v1/universes/{universeId}/standard-datastores",
-            universeId = universe_id,
-        )
+        format!("https://apis.roblox.com/datastores/v1/universes/{universe_id}/standard-datastores",)
     } else {
         format!(
-			"https://apis.roblox.com/datastores/v1/universes/{universeId}/standard-datastores/{endpoint}",
-			universeId=universe_id,
-			endpoint=endpoint,
+			"https://apis.roblox.com/datastores/v1/universes/{universe_id}/standard-datastores{endpoint}",
```

Comment:
This is the bugfix. Notice how the original formatter added a `/{endpoint}`, but all the other code that calls into `build_url` is prefixed with `/` too. So the `/` here was removed.